server/models/index.js

```diff
@@ -6,32 +6,23 @@ const Sequelize = require('sequelize');
 const basename = path.basename(__filename);
 const env = process.env.NODE_ENV || 'development';
 const config = require(__dirname + '/../config/config.js')[env];
-const db = {};
 
```

Comment:
Not sure how you intended to use the ternary here, but definitely let me know if there's a more conventional/preferred way to do it.

Reply:
Oh could you just do this?
```js
const sequelize = config.use_env_variable
  ? new Sequelize(process.env[config.use_env_variable], config)
  : new Sequelize(config.database, config.username, config.password, config);
```

Reply:
Cool, that was actually how I initially thought to write it - should've just went with my gut.

Updated.